shoot/schema_shoot.go

```diff
@@ -0,0 +1,919 @@
+package shoot
```

Comment:
A file with 919 lines of code is usually a big smell. Please split this into several, logically meaningful files and/or packages.

Reply:
done